smif/convert/interval.py

```diff
@@ -499,16 +511,8 @@ def convert(self, timeseries, from_interval, to_interval):
             total = 0
 
             for lower, upper in interval_tuples:
-
                 self.logger.debug("Range: %s-%s", lower, upper)
-
-                if upper < lower:
```

Comment:
I think this will break?  This conditional checks to see if a particular interval, expressed as a tuple of hourly positions between 0 and 8760, breaks over the start/end of a year.